command/build.py

```diff
@@ -1,8 +1,39 @@
+import yaml as pyYaml
+import command.identify as identify
+
 class BuildCmd():
     name = 'build'
-
-    def __init__(self):	
+    
+    def __init__(self):
         pass
+
+    def _get_empty_bundle(self):
+        operatorArtifactsTemplate = dict(
+            data = dict(
+                customResourceDefinitions = [],
+                clusterServiceVersions = [],
+                packages = [],
+            )
+        )
+        return operatorArtifactsTemplate
     
-    def build(self):
-        print("Hello, world. The build feature is currently under construction.")
+    def _get_field_entry(self, yamlContent):
+        yaml_type = identify.get_operator_artifact_type(yamlContent)
+        if yaml_type == "ClusterServiceVersion" or yaml_type == "CustomResourceDefinition" or yaml_type == "Package":
+            return yaml_type[0:1].lower() + yaml_type[1:] + 's'
```

Comment:
Do we need the if statement? The way this is used in `_updateArtifact` requires an output, so in the case that `yaml_type` is an unexpected value the program errors out; is this a "pythonic" way of erroring out? Should there be an error log as well?

Reply:
Actually the identify.get_operator_artifact_type method will raise an exception if the file isn't identified as a CSV, CRD, or Pacakge YAML - your points that the `if` statement isn't needed and logging are still valid.

Reply:
I don't think it makes sense to add a log when the raised exception will give the same information. I did address the comment related to removing the if statement --- we will never hit it.

Reply:
Addressed in latest commit.